### PR TITLE
Stabilize openshift 4.13.0; create openshift 4.14.0-experimental

### PR DESCRIPTION
**Pull request by prestist**
Roll back to Ignition 3.2.0 spec for #312.

### CODE REVIEW

File: config/openshift/v4_13/translate_test.go [hunk 1]
```diff
@@ -490,8 +403,7 @@ func TestValidateSupport(t *testing.T) {
 							},
 							Links: []base.Link{
 								{
-									Path:   "/l",
-									Target: util.StrToPtr("/t"),
```
Comment (bgilbert) [inline config/openshift/v4_13/translate_test.go]: Looks like this got dropped?

File: docs/config-openshift-v4_13.md [hunk 2]
```diff
@@ -1,24 +1,22 @@
 ---
-title: OpenShift v4.13.0-experimental
+title: OpenShift v4.13.0
 parent: Configuration specifications
 nav_order: 150
```
Comment (bgilbert) [inline docs/config-openshift-v4_13.md:4]: Needs update.

File: docs/config-openshift-v4_14-exp.md [hunk 3]
```diff
@@ -1,10 +1,10 @@
...
-title: OpenShift v4.13.0-experimental
+title: OpenShift v4.14.0-experimental
 parent: Configuration specifications
 nav_order: 150
 ---
 
-# OpenShift Specification v4.13.0-experimental
+# OpenShift Specification v4.14.0-experimental
 
 **Note: This configuration is experimental and has not been stabilized. It is subject to change without warning or announcement.**
```
Comment (bgilbert) [inline docs/config-openshift-v4_14-exp.md:10]: `version` field needs update.

File: docs/release-notes.md [hunk 4]
```diff
@@ -12,6 +12,9 @@ nav_order: 9
 
 ### Features
 
+- Stabilize OpenShift spec 4.13.0, targeting Ignition spec 3.2.0
+- Add OpenShift spec 4.14.0-experimental, targeting Ignition spec
+  3.4.0
 - Allow enabling discard passthrough on LUKS devices _(fcos 1.5.0-exp,
```
Comment (bgilbert) [inline docs/release-notes.md:18]: The `4.13.0-exp` release notes all need update to `4.13.0` or `4.14.0-exp`, depending on the item.

File: docs/config-openshift-v4_13.md (hunk 2 above)
Comment (prestist) [reply docs/config-openshift-v4_13.md:4]: Oh do we update the nav order each time? sorry I am not familiar with that, do I just ++ or somthing?

File: docs/release-notes.md (hunk 4 above)
Comment (prestist) [reply docs/release-notes.md:18]: Oh, ok

File: docs/config-openshift-v4_13.md (hunk 2 above)
Comment (prestist) [reply docs/config-openshift-v4_13.md:4]: Ah I see, so 144 would be appropriate, and I assume the exp spec has the highest number for the set?

Comment (bgilbert) [reply docs/config-openshift-v4_13.md:4]: It needs to be one less than the previous stable spec, same rule as a normal stabilization.

File: config/openshift/v4_13/translate_test.go (hunk 1 above)
Comment (prestist) [reply config/openshift/v4_13/translate_test.go]: Yeah, I accidentally dropped it. Thank you for catching that.

File: docs/release-notes.md (hunk 4 above)
Comment (prestist) [reply docs/release-notes.md:18]: So should I update the 3.4.0-exp notes to 3.4.0 as well while I am here? 
noticed they were still calling it exp

File: config/openshift/v4_14_exp/translate.go [hunk 5]
```diff
@@ -131,14 +131,14 @@ func (c Config) ToIgn3_4(options common.TranslateOptions) (types.Config, report.
 	return cfg.(types.Config), r, err
 }
 
-// ToConfigBytes translates from a v4.13 Butane config to a v4.13 MachineConfig or a v3.4.0 Ignition config. It returns a report of any errors or
+// ToConfigBytes translates from a v4.14 Butane config to a v4.13 MachineConfig or a v3.4.0 Ignition config. It returns a report of any errors or
```
Comment (bgilbert) [inline config/openshift/v4_14_exp/translate.go:134]:
```suggestion
// ToConfigBytes translates from a v4.14 Butane config to a v4.14 MachineConfig or a v3.4.0 Ignition config. It returns a report of any errors or
```

File: docs/release-notes.md [hunk 6]
```diff
@@ -12,12 +12,15 @@ nav_order: 9
...
-  flatcar 1.1.0-exp, openshift 4.13.0-exp)_
+  flatcar 1.1.0-exp, openshift 4.14.0-exp)_
 - Allow specifying arbitrary LUKS open options _(fcos 1.5.0-exp,
-  flatcar 1.1.0-exp, openshift 4.13.0-exp)_
-- Allow specifying user password hash _(openshift 4.13.0-exp)_
-- Support offline Tang provisioning via pre-shared advertisement _(fcos 1.5.0-exp, openshift 4.13.0-exp)_
+  flatcar 1.1.0-exp, openshift 4.14.0-exp)_
+- Allow specifying user password hash _(openshift 4.13.0)_
```
Comment (bgilbert) [inline docs/release-notes.md:22]: We need to also say that it's going into 4.14.0-exp, spelled _(openshift 4.13.0+)_.

File: docs/config-openshift-v4_13.md [hunk 7]
```diff
@@ -28,7 +26,7 @@ The OpenShift configuration is a YAML document conforming to the following speci
       * **_verification_** (object): options related to the verification of the config.
         * **_hash_** (string): the hash of the config, in the form `<type>-<value>` where type is either `sha512` or `sha256`. If `compression` is specified, the hash describes the decompressed config.
     * **_replace_** (object): the config that will replace the current.
-      * **_source_** (string): the URL of the config. Supported schemes are `http`, `https`, `s3`, `arn`, `gs`, `tftp`, and [`data`][rfc2397]. Note: When using `http`, it is advisable to use the verification option to ensure the contents haven't been modified. Mutually exclusive with `inline` and `local`.
+      * **_source_** (string): the URL of the config. Supported schemes are `http`, `https`, `s3`,`gs`, `tftp`, and [`data`][rfc2397]. Note: When using `http`, it is advisable to use the verification option to ensure the contents haven't been modified. Mutually exclusive with `inline` and `local`.
```
Comment (bgilbert) [inline docs/config-openshift-v4_13.md:29]:
```suggestion
      * **_source_** (string): the URL of the config. Supported schemes are `http`, `https`, `s3`, `gs`, `tftp`, and [`data`][rfc2397]. Note: When using `http`, it is advisable to use the verification option to ensure the contents haven't been modified. Mutually exclusive with `inline` and `local`.
````

File: docs/config-openshift-v4_13.md [hunk 8]
```diff
@@ -0,0 +1,168 @@
...
+    * **_mask_** (boolean): whether or not the service shall be masked. When true, the service is masked by symlinking it to `/dev/null`.
+    * **_contents_** (string): the contents of the unit.
+    * **_dropins_** (list of objects): the list of drop-ins for the unit. Every drop-in must have a unique `name`.
+      * **name** (string): the name of the drop-in. This must be suffixed with ".conf".
+      * **_contents_** (string): the contents of the drop-in.
+* **_passwd_** (object): describes the desired additions to the passwd database.
+  * **_users_** (list of objects): the list of accounts that shall exist. All users must have a unique `name`.
+    * **name** (string): the username for the account. Must be `core`.
+    * **_password_hash_** (string): the hashed password for the account.
+    * **_ssh_authorized_keys_** (list of strings): a list of SSH keys to be added to `.ssh/authorized_keys` in the user's home directory. All SSH keys must be unique.
```
Comment (bgilbert) [inline docs/config-openshift-v4_13.md:147]: Rebase error:

```suggestion
    * **_ssh_authorized_keys_** (list of strings): a list of SSH keys to be added to `.ssh/authorized_keys.d/ignition` in the user's home directory. All SSH keys must be unique.
```

**Review by bgilbert — COMMENTED** [review]
For something like this, diffing e.g. the 4.12 and 4.13 docs/packages is a useful cross-check.  It found a couple things.

**Review by bgilbert — APPROVED** [review]

**Comment by bgilbert** [comment]
Followup in #440.

**Pull request merged**